lib/flipper/api/v1/actions/boolean_gate.rb

```diff
@@ -0,0 +1,21 @@
+require 'flipper/api/action'
+
+module Flipper
+  module Api
+    module V1
+      module Actions
+        class BooleanGate < Api::Action
+          route %r{api/v1/features/[^/]*/(enable|disable)/?\Z}
+          
+          def put
+            feature_name = Rack::Utils.unescape(request.path.split("/")[-2])
+            feature = flipper[feature_name.to_sym]
+            action = Rack::Utils.unescape(request.path.split("/").last)
+            feature.send(action)
```

Comment:
OIC. You only match enable/disable in route so it is safe by those means. I guess that seems ok.